apps/s_server.c

```diff
@@ -1741,6 +1759,10 @@ int s_server_main(int argc, char *argv[])
     if (async) {
         SSL_CTX_set_mode(ctx, SSL_MODE_ASYNC);
     }
+
+    if (max_send_fragment > 0)
+        SSL_CTX_set_max_send_fragment(ctx, max_send_fragment);
```

Comment:
The returned value should be checked.  0 indicates failure...  and really, that should also be done for the three calls below as well...